tests/shared/src/test/scala/scala/meta/tests/tokenizers/TokenizerSuite.scala

```diff
@@ -1170,7 +1170,7 @@ class TokenizerSuite extends BaseTokenizerSuite {
   test("#3328") {
     val code = "val \\uD835\\uDF11: Double"
     val res = dialects.Scala212(code).tokenize
-    assertEquals(res.toEither.left.get.message, "illegal character '\\ud835'")
```

Comment:
Could you compare the tokens explicitely so that we can easily see how it's tokenized?

Reply:
added another test case